indexing-service/src/main/java/io/druid/indexing/common/index/RealtimeAppenderatorTuningConfig.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.druid.indexing.common.index;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.google.common.base.Preconditions;
+import com.google.common.io.Files;
+import io.druid.segment.IndexSpec;
+import io.druid.segment.indexing.TuningConfig;
+import io.druid.segment.realtime.appenderator.AppenderatorConfig;
+import io.druid.segment.writeout.SegmentWriteOutMediumFactory;
+import io.druid.timeline.partition.NoneShardSpec;
+import io.druid.timeline.partition.ShardSpec;
+import org.joda.time.Period;
+
+import javax.annotation.Nullable;
+import java.io.File;
+
+@JsonTypeName("appenderator")
```

Comment:
What do you suggest? _realtime_appenderator_ ?

Reply:
I suggest `index_realtime_appenderator` (https://github.com/druid-io/druid/pull/5261/files#diff-df997e60df22e57ef92bff9af75fa9d5R58). This is because users will be easier to remember if they use the same name.
Or I'm ok with changing both type names to `realtime_appenderator`.

Reply:
Do you think there is value in keeping the same pattern as the existing realtime task?

As of now, the `RealtimeIndexTask` task ID is _index_realtime_ with a corresponding tuningConfig value of _realtime_.  If I kept the same pattern for this new task, the ID would be _index_realtime_appenderator_ (as it currently is) and I would change the tuningConfig to be _realtime_appenderator_.

Thoughts?

Reply:
Sounds good. It looks better to keep the current pattern.

Reply:
No problem, updated.

Reply:
Thanks for the quick fix!